substrate/cmd/bootstrap.go

```diff
@@ -25,18 +26,30 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-func init() {
-	rootCmd.AddCommand(&cobra.Command{
-		Use:   "apply",
-		Short: "Apply an environment for testing. Will reconnect if the environment already exists.",
+func bootstrapCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "bootstrap",
```

Comment:
Should we just use the terms create/delete? Since we're creating/deleting an env? Or do we imagine many other commands in kitcli? My assumption is that you should interact with guest clusters using kubectl.

Reply:
I plan on adding a `create` command for cluster creation, that would generate a control-plane spec. Something like `kitctl create cluster foo`

Reply:
This will have strange parity with delete, since create and delete aren't inverted from each other.

Reply:
Yeah I noticed that as well, this is what I was thinking the flow will look like, wdyt?
```
kictl bootstrap
kictl create cluster foo
kictl delete cluster foo
kitctl cleanup
```